<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_set_mashen</class>
 <widget class="QDialog" name="Dlg_set_mashen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>麻醉深度设置</string>
  </property>
  <widget class="QComboBox" name="m_cmbb_ap">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>7</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_cancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ok">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_speed">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>脑电波幅度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>波形速度</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
